index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TEDxUniversity</title>
+    <!-- link to external resources -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <!-- link to other files -->
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="js/main.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <div class="d-navbar flex g-padding">
+        <a href="#">
+          <img class="logo" src="images/tedx-logo.jpg" alt="TEDx's Logo" />
+        </a>
+        <nav class="d-nav">
```

Comment:
- [ ] The top dark menu section missing on both pages. Please implement this section and be sure : 

- The social media links, logout, and language links are aligned to the right.
- Section should disappear on the mobile resolution.

 ![Screenshot from 2021-09-09 22-19-47](https://user-images.githubusercontent.com/1642014/132749370-3ceac04f-19ea-49fa-921f-05929662ed93.png)